lnst/Recipes/ENRT/VhostNetPvPRecipe.py

```diff
@@ -83,8 +85,9 @@ def test_wide_configuration(self, config):
         config.generator.nics.append(self.matched.generator_req.eth0)
         config.generator.nics.append(self.matched.generator_req.eth1)
 
-        self.matched.generator_req.eth0.ip_add(ipaddress("192.168.1.1/24"))
-        self.matched.generator_req.eth1.ip_add(ipaddress("192.168.1.2/24"))
+        ipv4_addr = interface_addresses(self.params.net_ipv4)
+        self.matched.generator_req.eth0.ip_add(next(ipv4_addr))
+        self.matched.generator_req.eth1.ip_add(next(ipv4_addr))
```

Comment:
fyi, for these dpdk related recipes we may also need to update the base class that does a "warmup" ping: https://github.com/pgagne/lnst/blob/45a862b9bf7d7666a14d6791c145ca60ec660f44/lnst/Recipes/ENRT/BasePvPRecipe.py#L105

i think the warmup is actually not really related so maybe we can also completely remove it, i think the idea i started with may not have been fully functional even though the thought behind it is valid - warmup here should fill up the switch forwarding table but from what i noticed, it gets automatically dumped anyway when we proceed with the dpdk reconfiguration that we do after the warmup.

Reply:
Will remove warmup code.